src/sentry/static/sentry/app/components/events/interfaces/rawStacktraceContent.jsx

```diff
@@ -150,14 +163,22 @@ export default function render (data, platform, exception) {
   }
 
   data.frames.forEach((frame, frameIdx) => {
-    children.push(getFrame(frame, platform));
+    frames.push(getFrame(frame, platform));
     if (frameIdx === firstFrameOmitted) {
-      children.push((
+      frames.push((
         '.. frames ' + firstFrameOmitted + ' until ' + lastFrameOmitted + ' were omitted and not available ..'
       ));
     }
 
   });
 
-  return children.join('\n');
+  if (platform !== 'python') {
```

Comment:
This follows the precedent established here, although I'm not totally sure it is correct: https://github.com/getsentry/sentry/blob/6d9a1ef092c5b5ef709da8fd1f59b5757aa89e55/src/sentry/static/sentry/app/components/events/interfaces/exception.jsx#L31-L32

Since this is emulating that behavior, it might make sense to make the ordering by platform a constant somewhere?